src/components/cards/video-card/VideoCard.jsx

```diff
@@ -32,10 +33,12 @@ const VideoCard = ({
   return (
     <>
       <div className="video-card-container card-pos-rel">
-        {history && (
-          <MdDelete
-            className="video-card-icon cursor color-red"
-            onClick={removeFromHistory}
+        {!watchLater && (
+          <AiFillLike
+            className={`video-card-icon cursor ${
+              liked ? "color-green" : "color-black"
```

Comment:
Feel like the class names here can be a little more generic like liked-video-card